operations/image_management/Convert_TGZ_Archives_to_SquashFS_Images.md

```diff
@@ -10,24 +10,23 @@ There is a pre-built image that is not currently in SquashFS format.
 
 ## Procedure
 
-1.  Locate the image root to be converted to SquashFS.
+1. Locate the image root to be converted to SquashFS.
 
     Images and recipes are uploaded to IMS and S3 via containers.
 
-2.  Uncompress the desired file to a temporary directory.
+1. Uncompress the desired file to a temporary directory.
 
-    Replace the TXZ\_COMPRESSED\_IMAGE value with the name of the image root being used that was returned in the previous step.
+    Replace the `TXZ_COMPRESSED_IMAGE` value with the name of the image root being used that was located in the previous step.
 
     ```bash
-    ncn-m001# mkdir -p ~/tmp/image-root
-    ncn-m001# cd ~/tmp/
-    ncn-m001# tar xvf TXZ_COMPRESSED_IMAGE -C image-root
+    ncn# mkdir -p ~/tmp/image-root
```

Comment:
This section I am not sure would work on a storage node either.  lines 22-31.

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815